class/systems.class.php

```diff
@@ -190,10 +190,37 @@ public function getSystemActivities($system)
 		$this->db->bind(':system', $system);
 		$result = $this->db->resultset();
 		$this->db->closeQuery();
-		
+
 		return $result;
 	}
 
+		/**
+	 * Get participants of currently active cache
+	 * @param unknown $system
+	 * @return string
+	 */
+	public function getActiveCacheParticipants($system)
+	{
+			$queryString = "SELECT Distinct Pilot
+					FROM activity
+					WHERE System = :system
+					AND ActivityDate >= (SELECT ActivityDate
+							FROM activity
+							WHERE System = :systemII
+							And EntryType = 'sower'
+							ORDER By ActivityDate ASC
+							LIMIT 1) ORDER By ActivityDate ASC";
```

Comment:
This is great stuff Klensor. Thank you.
I think I would like to introduce this email system with some extra niceties.  
For example, the email should really show the dates the pilots last tended it. I think (if I were a pilot) I would love to see that a cache I tended 3 months ago is still alive and helped someone!

to do that I wrote you this query, which should do the job 

```
SELECT Pilot, EntryType, DATE_FORMAT(MAX(ActivityDate), '%b-%d-%Y') LastActivity
		FROM activity
		WHERE System = 'J135036'
		AND ActivityDate >=
			(SELECT MAX(ActivityDate) as LastActivity
				FROM activity
				WHERE System = 'J135036'
				AND EntryType = 'sower')
GROUP BY Pilot
ORDER BY MAX(ActivityDate)
```

this query executes in .0006 seconds  - its fine

It returns this for J135036

Pilot | EntryType | LastActivity 
Aldar Roanaok | sower | Jul-28-2019
Bang N' Donk | tender | Aug-03-2019
Tamayo | tender | Aug-15-2019
Troubled Watters | tender | Aug-24-2019
Nac Audene | tender | Sep-17-2019